glib/src/object.rs

```diff
@@ -1858,7 +1865,6 @@ pub trait ObjectExt: ObjectType {
     ) -> SignalHandlerId;
 
     /// Similar to [`Self::connect_closure_id`] but fails instead of panicking.
-    #[doc(alias = "g_signal_connect_closure")]
```

Comment:
Why? I would add both aliases here too

Reply:
I can add them back, but imho it was weird to have them on the `try_` variant

Reply:
Why? It does exactly the same, just returns a result instead of panicking

Reply:
well, then we should add them to all the other `try` variants as well, this was the one-off... I can do that

Reply:
Oh I see. Then let's not do that for now and add them all at once later if needed. Thanks for noticing that :)